src/capi/azext_capi/custom.py

```diff
@@ -520,11 +531,114 @@ def create_workload_cluster(  # pylint: disable=unused-argument,too-many-argumen
 
     # Wait for all nodes to be ready before returning
     with Spinner(cmd, "Waiting for workload cluster nodes to be ready", "✓ Workload cluster is ready"):
-        wait_for_ready(workload_cfg)
+        wait_for_nodes(workload_cfg)
 
+    if pivot:
+        pivot_cluster(cmd, workload_cfg)
     return show_workload_cluster(cmd, capi_name)
 
 
+def pivot_cluster(cmd, target_cluster_kubeconfig):
+
+    logger.warning("Starting Pivot Process")
+    begin_msg = "Installing CAPZ components in target management cluster"
+    end_msg = "✓ Installed CAPZ components in target management cluster"
+    with Spinner(cmd, begin_msg, end_msg):
+        set_azure_identity_secret_env_vars()
+        _create_azure_identity_secret(cmd, target_cluster_kubeconfig)
+        _install_capz_components(cmd, target_cluster_kubeconfig)
+
+    with Spinner(cmd, "Waiting for workload cluster machines to be ready", "✓ Workload cluster machines are ready"):
+        wait_for_machines()
+
+    command = ["clusterctl", "move", "--to-kubeconfig", target_cluster_kubeconfig]
+    begin_msg = "Moving cluster objects into target cluster"
+    end_msg = "✓ Moved cluster objects into target cluster"
+    error_msg = "Could not complete clusterctl move action"
+    try_command_with_spinner(cmd, command, begin_msg, end_msg, error_msg, True)
+
+    cluster_name = find_cluster_in_current_context()
+    if has_kind_prefix(cluster_name):
+        delete_kind_cluster_from_current_context(cmd)
+    else:
+        resource_group = os.environ.get(MANAGEMENT_RG_NAME, None)
+        if not resource_group:
+            raise UnclassifiedUserFault("Could not delete aks management cluster, resource group missing")
```

Comment:
```suggestion
            raise UnclassifiedUserFault("Could not delete AKS management cluster, resource group missing")
```

We should capitalize AKS when referring to the service in user-visible text, to be consistent with Azure docs.